thoth/common/openshift.py

```diff
@@ -1308,22 +1292,9 @@ def verify_kebechet_inputs(
     def verify_integration_inputs(
         self,
         source_type: Optional[ThothAdviserIntegrationEnum],
-        github_event_type: Optional[str] = None,
-        github_check_run_id: Optional[int] = None,
-        github_installation_id: Optional[int] = None,
-        github_base_repo_url: Optional[str] = None,
```

Comment:
These need to be removed from schedule_adviser below I believe

Reply:
Thanks!